### ISSUE TITLE
Loading trained models give unexpected result.

### ISSUE THREAD

**Issue by lsbmsb** [pytorch/examples#505]
I have modified the example from examples/blob/master/time_sequence_prediction/train.py for my use for the prediction of time series. I have trained the models in different settings of hyperparameters to choose the best one and saved it with torch.save(). When I load the model with torch.load() first time, it gives correct test loss as I check at the time of the training but subsequent load for same model loads different weights giving erroneous output. Here is a complete issue from pytorch forums : https://discuss.pytorch.org/t/loading-multiple-saved-models-gives-unexpected-results/36312/11

**Comment by lsbmsb** [comment]
Resolved the issue by saving the separate dicts of multiple LSTM cells and loading them individually,
https://discuss.pytorch.org/t/loading-saved-models-gives-inconsistent-results-each-time/36312/24
